Makefile

```diff
@@ -166,3 +165,11 @@ bedrock-markdown-links:
 
 install-geth:
 	go install github.com/ethereum/go-ethereum/cmd/geth@v1.12.0
+
+generate-hotshot-binding:
+	forge build --root espresso-sequencer --out ../out --extra-output-files abi
```

Comment:
Do we need to use `--force` here? I don't really understand forge's caching but seems like sometimes this is necessary

Reply:
yeah force isn't necessary here because we are using a transient output file